subsys/testsuite/Kconfig

```diff
@@ -97,6 +97,16 @@ config FORCE_COVERAGE
 	  does not guarantee that coverage data will be gathered.
 	  Application may not fit memory or crash at runtime.
 
+config COVERAGE_USE_COMPILER_FLAGS
```

Comment:
Please add a section about this in the documentation and show how this will be used, i.e. an example of how you can build code with the compiler options applied. There is a documentation page about coverage where this can be added: develop/test/coverage.rst

Reply:
@nashif I see that this page doesn't cover any of the available options except CONFIG_COVERAGE. I am not sure I have a simple explanation that will be useful for many people. We have a very custom build pipeline that allows compiling some code separately and then linking it to the zephyr binary. I did my best to explain that in the Kconfig doc

Reply:
I guess this just need an additional entry to https://docs.zephyrproject.org/latest/develop/test/coverage.html ? The text in this `help` section can probably be reused.